arch/arm64/boot/dts/qcom/pmi8950.dtsi

```diff
@@ -206,10 +206,9 @@
 			qcom,current-limit-microamp = <20000>;	// qcom,fs-curr-ua
 			qcom,ovp-millivolt = <29600>;		// qcom,ovp-mv
 			qcom,switching-freq = <800>;		// qcom,switch-freq-khz
-			qcom,enabled-strings = <0 1 2 3>;	// qcom,led-strings-list
```

Comment:
I'm not sure if this is okay, maybe other devices rely on this being set.

Reply:
a warning in the dmesg for 5.18 says clearly that it's a duplicated, you can go only for one of them